mindsdb/interfaces/native/predictor_process.py

```diff
@@ -38,7 +38,7 @@ def run(self):
                 'name': name,
                 'predict_cols': to_predict,
                 'data_analysis': stats
-            }])
+            }], setup = False)
```

Comment:
I believe this needs to be set to `True` here for now, I'm afraid.

As in, assuming a new connection is added, we need to register the prediction to it, unless the connections are re-established every time, we can't (at the moment) make that happen. Am I wrong in assuming this ?

Reply:
This block should not executed if connection added. It executed just when predictor trained. And there was error, what was fixed by adding 'setup = False':
1. user start learn predictor by send to db statement 'insert into mindsdb.predictors ....'
2. when learning finished, then this 'DatabaseWrapper.register_predictors' starts. 
3. before 'setup' parameter was added, it each time execute '_setup_integrations' what follows to 'integration.setup' where executed 'drop database if exists mindsdb', what follow to db lock.